public/js/script.js

```diff
@@ -0,0 +1,204 @@
+async function handlePartnerLogin() {
```

Comment:
would have preferred you don't use JS for anything else than basic animations; functionality already good enough if server-client alone; however, even like that, injecting DOM elements is code smell